tests/test_compile.py

```diff
@@ -34,7 +34,7 @@ def setUp(self):
         os.chdir(os.getcwd() + '/examples/kubernetes/')
 
     def test_compile(self):
-        sys.argv = ["kapitan", "compile", "-c"]
+        sys.argv = ["kapitan", "compile", "--cache"]
```

Comment:
This doesn't change anything

Reply:
python-fire renders cli on basis of argument names supplied in function signature and so earlier only --cache used to work (as it was supplied in function argument) but later (refer to https://github.com/google/python-fire/issues/108 and https://github.com/google/python-fire/pull/141 ) the shorthand notation ability is indeed added into fire. But the released version on pypi lacks that ability which is soon going to be updated so we can have the shorthand test cases updated later. Also note that capital flags(like -F) are still not allowed and Fire raises error and tell the user in case one uses shorthand notation for two flags whose first letter is same and in that case one has to explicitly write the whole name to avoid conflict.

Reply:
Ok, can you open an issue on python-fire for them to update the release and we change things after.